src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -268,14 +268,14 @@ debug_configuration::debug_configuration()
         dump_prof_data_iter_str = " " + dump_prof_data_iter_str + " ";
         std::istringstream iss(dump_prof_data_iter_str);
         char dot;
-        int start, end;
+        int64_t start, end;
         bool is_valid_range = false;
         if (iss >> start >> dot >> dot >> end) {
-            if (start < end) {
+            if (start <= end || end == -1) {
```

Comment:
Please add to the help message regarding the open ended range (N, -1) is available too.

Reply:
help message has been also updated